cocoa/booking.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+from model import ModelService
+from booking import BookingService
```

Comment:
Line 3 : from notifier import NotifService

Reply:
we shouldn't be importing notifier service here as notifier service will import booking service, which will create circular dependencies

Reply:
@BALaka-18 since this is a PR against the dev branch, lets merge it for now. can iron out the kinks and then merge it into main

Reply:
> we shouldn't be importing the notifier service here as the notifier service will import this service creating a circular dependency

Ya even I thought so too, but then saw that he was using the notifier service in line 8, so I thought maybe I was getting it wrong or something. Since it's merged now, I'll resolve this conversation.